src/04-modality-specific-files/11-functional-near-infrared-spectroscopy.md

```diff
@@ -56,35 +56,39 @@ be directly attached to the scalp with adhesive. To facilitate description of
 the wide variety of possible configurations, several fields are RECOMMENDED within
 the `*_nirs.json` file.
 Additionally, in certain situations, reserved keywords MUST be used.
-When a custom made cap is used, the reserved keyword `custom` MUST be used in the
-`CapManufacturer` field.
-Similarly, when a custom cap is used or custom modifications are made to a cap,
+When custom modifications are made to a commercially available cap or a custom cap is used,
 then the reserved keyword `custom` MUST be used for the `CapManufacturersModelName` field.
+When a custom made cap is used, that is, no (modified) commercially available cap,
+the reserved keyword `custom` MUST be used in the `CapManufacturer` field.
 If no cap is used, the reserved keyword `none` MUST be used in the `CapManufacturer`
 and `CapManufacturersModelName` field.
+The use of `NIRSPlacementScheme` is RECOMMENDED when no cap or a customized cap is used,
+and describes the positioning of the optodes. This field may also contain a reference to a file providing
+a graphical depiction of the cap, for example a PDF file, a photo, or a bitmap drawing. Note that this file
+should be placed in the [`/sourcedata`](../02-common-principles.md#source-vs-raw-vs-derived-data) directory, as it cannot be recognized by the validator.
```

Comment:
Here's some language that leaves room for us to specify some options in the future:

```suggestion
and describes the positioning of the optodes.
This field may also contain a reference to a file providing a graphical depiction of the cap,
for example a PDF file, a photo, or a bitmap drawing.
If the referred file is not specified in BIDS, it MAY be placed in the
[`/sourcedata`](../02-common-principles.md#source-vs-raw-vs-derived-data) directory.
```

I don't want to suggest adding it to the BEP at this late date, but *EG and microscopy all define the `_photo` suffix, which might cover the use cases that don't require a PDF containing multiple images. That could be an easy extension after 1.8.0.

Reply:
A difference is that in \*EG the "landmark and/or fiducial" photos are taken at the individual participant level and serve as "data", whereas here it (probably) is a one-time photo of a cap on a dummy head or a schematic drawing that serves to document the cap design.

So summarizing: we have photos for the purpose of \*EG coregistration, and we can have a  picture or drawing of the NIRS cap. We could in principle define/document both the other way around as well (i.e. custom EEG or MEG-OPM cap design picture, and NIRS landmark photos), but I don't see the immediate need for that. I suggest to do that with a future small PR once the need arises (i.e. upon request).